R/Driver.R

```diff
@@ -37,48 +37,63 @@ setMethod(
     # TODO: Print more details
   })
 
-#' Connect to a ODBC compatible database
+#' Connect to an ODBC compliant database
 #'
-#' @inheritParams DBI::dbConnect
-#' @param dsn The Data Source Name.
-#' @param timezone The Server time zone. Useful if the database has an internal
-#' timezone that is _not_ 'UTC'. If the database is in your local timezone set
-#' to [Sys.timezone()]. See [OlsonNames()] for a complete list of available
-#' timezones on your system.
+#' This method is invoked when [DBI::dbConnect()] is called with the first argument
+#' [odbc::odbc()].
+#'
+#' @param drv An `OdbcDriver` object, likely outputted from [odbc::odbc()].
+#' @param dsn The data source name. For currently available options, see the
+#' `name` column of [odbcListDataSources()] output.
+#' @param timezone The server time zone. Useful if the database has an internal
+#' timezone that is _not_ 'UTC'. If the database is in your local timezone, set
+#' this argument to [Sys.timezone()]. See [OlsonNames()] for a complete list of
+#' available timezones on your system.
 #' @param timezone_out The time zone returned to R. If you want to display
-#' datetime values in the local timezone, set to `Sys.timezone()`.
+#' datetime values in the local timezone, set to [Sys.timezone()].
 #' @param encoding The text encoding used on the Database. If the database is
 #' not using UTF-8 you will need to set the encoding to get accurate re-encoding.
 #' See [iconvlist()] for a complete list of available encodings on your system.
 #' Note strings are always returned `UTF-8` encoded.
-#' @param driver The ODBC driver name.
-#' @param server The server hostname.
-#' @param database The database on the server.
-#' @param uid The user identifier.
-#' @param pwd The password to use.
+#' @param driver The ODBC driver name. For currently available options, see the
```

Comment:
Can be either a driver name or a path to a driver.

Reply:
[5afcce0](https://github.com/r-dbi/odbc/pull/647/commits/5afcce05f343840718fa13ef880b53741e295070).